loopdb/store.go

```diff
@@ -99,9 +99,18 @@ var (
 	keyLength = 33
 )
 
-// DefaultLoopOutHtlcConfirmations is the default number of confirmations we
-// set for a loop out htlc.
-const DefaultLoopOutHtlcConfirmations uint32 = 1
+const (
+	// DefaultLoopOutHtlcConfirmations is the default number of
+	// confirmations we set for a loop out htlc.
+	DefaultLoopOutHtlcConfirmations uint32 = 1
+
+	// DefaultLoopDBTimeout is the default maximum time we wait for the
+	// Loop bbolt database to be opened. If the database is already opened
+	// by another process, the unique lock cannot be obtained. With the
+	// timeout we error out after the given time instead of just blocking
+	// for forever.
+	DefaultLoopDBTimeout = 5 * time.Second
```

Comment:
there's also the `macDatabaseOpenTimeout`, should we unify them and use only `DefaultLoopDBTimeout` instead? Also, just a thought, maybe consider making this configurable in the future?

Reply:
Ah, good point. I removed the `macDatabaseOpenTimeout` and replaced it with `DefaultLoopDBTimeout`.
I'm not sure how much value customizing this timeout would give us at the moment, so I'm leaving it out of this PR. Should be pretty easy to add if we ever see a use case.